bin/score.py

```diff
@@ -19,9 +20,31 @@ def score_submission(predictions_path: str, status: str) -> dict:
         score_status = "INVALID"
         score1, score2, score3 = None, None, None
     else:
+        # Unzipping the predictions and extracting the files in
+        # the current working directory
+        if ".zip" in os.path.basename(predictions_path):
```

Comment:
Consider swapping to use https://docs.python.org/3/library/zipfile.html#zipfile.is_zipfile

Reply:
Thanks! Didn't know about this. But I wonder if it's best to leave it like this since explicitly looking for `".zip"` appears to be more robust that `is_zipfile`. Here's a snippet where I was trying it out and it doesn't recognize the zipfle when there was a space at the beginning of the string:

```
In [1]: file =" /Users/jmedina/Downloads/Archive.zip"

In [2]: import zipfile

In [3]: zipfile.is_zipfile
Out[3]: <function zipfile.is_zipfile(filename)>

In [4]: zipfile.is_zipfile(file)
Out[4]: False
```